.github/workflows/CI-for-aarch64.yml

```diff
@@ -0,0 +1,142 @@
+---
+name: CI for Aarch64
+
+on:
+  pull_request:
+    branches: [master, nightly]
+    types: [opened, synchronize, reopened]
+  push:
+    branches: [master]
+  workflow_dispatch:
+
+jobs:
+  github_env:
+    name: GitHub Env Debug
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Dump github context
+        run: echo "$GITHUB_CONTEXT"
+        shell: bash
+        env:
+          GITHUB_CONTEXT: ${{ toJson(github) }}
+
+  check_changelog:
+    name: Check Changelog
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Verify Changelog
+        id: verify_changelog
+        if: ${{ github.ref == 'refs/heads/master' || github.base_ref == 'master' }}
+        # base_ref for pull request check, ref for push
+        uses: LizardByte/.github/actions/verify_changelog@master
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+    outputs:
+      next_version: ${{ steps.verify_changelog.outputs.changelog_parser_version }}
+      next_version_bare: ${{ steps.verify_changelog.outputs.changelog_parser_version_bare }}
+      last_version: ${{ steps.verify_changelog.outputs.latest_release_tag_name }}
+      release_body: ${{ steps.verify_changelog.outputs.changelog_parser_description }}
+
+  check_versions:
+    name: Check Versions
+    runs-on: ubuntu-latest
+    needs: check_changelog
+    if: ${{ github.ref == 'refs/heads/master' || github.base_ref == 'master' }}
+    # base_ref for pull request check, ref for push
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Check CMakeLists.txt Version
+        run: |
+          version=$(grep -o -E '^project\(Sunshine VERSION [0-9]+\.[0-9]+\.[0-9]+' CMakeLists.txt | \
+            grep -o -E '[0-9]+\.[0-9]+\.[0-9]+')
+          echo "cmakelists_version=${version}" >> $GITHUB_ENV
+
+      - name: Compare CMakeList.txt Version
+        if: ${{ env.cmakelists_version != needs.check_changelog.outputs.next_version_bare }}
+        run: |
+          echo CMakeLists version: "$cmakelists_version"
+          echo Changelog version: "${{ needs.check_changelog.outputs.next_version_bare }}"
+          echo Within 'CMakeLists.txt' change "project(Sunshine [VERSION $cmakelists_version]" to \
+            "project(Sunshine [VERSION ${{ needs.check_changelog.outputs.next_version_bare }}]"
+          exit 1
+
+  build_linux_flatpak_aarch64:
```

Comment:
I think this should do it (modifying the original CI file).

```yml
build_linux_flatpak:
    name: Linux Flatpak
    runs-on: ubuntu-22.04
    needs: check_changelog
    strategy:
      fail-fast: false  # false to test all, true to fail entire job if any fail
      matrix:
        include:  # package these differently
          - arch: x86_64
          - arch: aarch64

    steps:
      - name: Checkout
        uses: actions/checkout@v3

      - name: Setup Dependencies Linux Flatpak
        run: |
          sudo apt-get update -y
          sudo apt-get install -y \
            cmake \
                              
            flatpak
          sudo su $(whoami) -c 'flatpak remote-add --user remote-add --if-not-exists flathub \
            https://flathub.org/repo/flathub.flatpakrepo'
          sudo su $(whoami) -c 'flatpak --user install -y flathub \
            org.flatpak.Builder \
            org.freedesktop.Platform/${{ matrix.arch }}/21.08 \
            org.freedesktop.Sdk/${{ matrix.arch }}/21.08'
      - name: Configure Flatpak Manifest
        run: |
          # variables for manifest
          branch=${GITHUB_HEAD_REF}
          # check the branch variable
          if [ -z "$branch" ]
          then
            echo "This is a PUSH event"
            branch=${{ github.ref_name }}
            commit=${{ github.sha }}
            clone_url=${{ github.event.repository.clone_url }}
          else
            echo "This is a PR event"
            commit=${{ github.event.pull_request.head.sha }}
            clone_url=${{ github.event.pull_request.head.repo.clone_url }}
          fi
          echo "Branch: ${branch}"
          echo "Commit: ${commit}"
          echo "Clone URL: ${clone_url}"
          mkdir -p build
          mkdir -p artifacts
          cd build
          cmake -DGITHUB_CLONE_URL=${clone_url} \
            -DGITHUB_BRANCH=${branch} \
            -DGITHUB_COMMIT=${commit} \
            -DSUNSHINE_CONFIGURE_FLATPAK_MAN=ON \
            -DSUNSHINE_CONFIGURE_ONLY=ON \
            ..
      - name: Build Linux Flatpak
        working-directory: build
        run: |
          sudo su $(whoami) -c 'flatpak run org.flatpak.Builder --arch=${{ matrix.arch }} --repo=repo --force-clean build-sunshine \
            dev.lizardbyte.sunshine.yml'
          sudo su $(whoami) -c 'flatpak build-bundle --arch=${{ matrix.arch }} ./repo ../artifacts/sunshine_${{ matrix.arch }}.flatpak dev.lizardbyte.sunshine'
      - name: Upload Artifacts
        uses: actions/upload-artifact@v3
        with:
          name: sunshine-linux-flatpak-${{ matrix.arch }}
          path: artifacts/

      - name: Create Release
        if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/master' }}
        uses: LizardByte/.github/actions/create_release@master
        with:
          token: ${{ secrets.GH_BOT_TOKEN }}
          next_version: ${{ needs.check_changelog.outputs.next_version }}
          last_version: ${{ needs.check_changelog.outputs.last_version }}
          release_body: ${{ needs.check_changelog.outputs.release_body }}
```

Reply:
I updated the yml above. This should only run the `aarch64` matrix when the release event happens. The update is based on this answer (although not highly voted) https://stackoverflow.com/a/73822998/11214013

This is untested, but we can get an idea if it works just by updating the CI file. The `aarch64` should not run on PR events.